climada_petals/hazard/landslide.py

```diff
@@ -37,43 +37,88 @@
 LOGGER = logging.getLogger(__name__)
 HAZ_TYPE = 'LS'
 
-def sample_events_from_probs(prob_matrix, n_years, dist='binom'):
-    """sample an event set for a specified representative time span from
-    a matrix with annual occurrence probabilities
-    Draws events from chosen distribution.
 
+def sample_events(prob_matrix, n_years, dist='binom'):
+    """
+    Sample yearly events for a specified time span (n_years) from
+    a matrix representing annual occurrence probabilities per grid cell 
+    (prob_matrix). The matrix will usually be provided throught the
+    Landslide.from_prob() method and refer to grid-cells and their annual 
+    sliding probabilities.
+    Events are drawn from the specified distribution (binomial or 
+    poisson). 
+    Returns n_year events for the whole grid, each event being representative
+    of all slides happening in 1 year throughout the entire grid-area.
```

Comment:
Much improved docstring! Yet, I do not understand what an event is... Is an event a full matrix?

Reply:
yes, one event refers to all the 0/1s over all grid cells being sampled for that year.